cmd/experimental/kjobctl/pkg/cmd/create/create.go

```diff
@@ -138,15 +154,31 @@ var createModeSubcommands = map[string]modeSubcommand{
 	"interactive": {
 		ModeName: v1alpha1.InteractiveMode,
 		Setup: func(subcmd *cobra.Command, o *CreateOptions) {
-			subcmd.Use += " [--pod-running-timeout DURATION] [--rm]"
+			subcmd.Use += " [--cmd COMMAND] [--pod-running-timeout DURATION] [--rm]"
 			subcmd.Short = "Create an interactive shell"
 			subcmd.Example = createInteractiveExample
+			subcmd.Flags().StringVar(&o.UserSpecifiedCommand, commandFlagName, "",
+				"Command which is associated with the resource.")
 			subcmd.Flags().DurationVar(&o.PodRunningTimeout, podRunningTimeout, podRunningTimeoutDefault,
 				"The length of time (like 5s, 2m, or 3h, higher than zero) to wait until at least one pod is running.")
 			subcmd.Flags().BoolVar(&o.RemoveInteractivePod, "rm", false,
 				"Remove pod when interactive session exits.")
 		},
 	},
+	"rayjob": {
+		ModeName: v1alpha1.RayJobMode,
+		Setup: func(subcmd *cobra.Command, o *CreateOptions) {
+			subcmd.Use += " [--replicas [WORKER_GROUP]=REPLICAS] [--min-replicas [WORKER_GROUP]=MIN_REPLICAS] [--max-replicas [WORKER_GROUP]=MAX_REPLICAS]"
+			subcmd.Short = "Create a rayjob"
+			subcmd.Example = createRayJobExample
+			subcmd.Flags().StringToIntVar(&o.Replicas, replicasFlagName, nil,
+				"Replicas is the number of desired Pods for this worker group.")
+			subcmd.Flags().StringToIntVar(&o.MinReplicas, minReplicasFlagName, nil,
+				"MinReplicas denotes the minimum number of desired Pods for this worker group.")
+			subcmd.Flags().StringToIntVar(&o.MaxReplicas, maxReplicasFlagName, nil,
+				"MaxReplicas denotes the maximum number of desired Pods for this worker group, and the default value is maxInt32.")
+		},
```

Comment:
We should keep the `cmd` flag for ray as well, a job user ca chose to run a different file from a volume mount provided by the profile.

Without this, just being able run the profile with different replicas is a very limiting UX.

Reply:
Do you mean entrypoint? 

https://github.com/kubernetes-sigs/kueue/blob/2d971cff71a8d4ac3bcddf7c0f955b24c6ceb704/cmd/experimental/kjobctl/config/samples/application_profile.yaml#L63

Reply:
Yes, but for the create command uniformity let's keep naming it cmd